### PR TITLE
melt_czochralski_Dadzis: simplify schema

### PR DESCRIPTION
**Pull request by arvedes**
I simplified the main eln file by removing sensors_list and sensor_locations and removed some unnecessary plots from the DAQ-schema. Furthermore, I fixed inconsistent indentation.

### CODE REVIEW

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml [hunk 1]
```diff
@@ -2,51 +2,51 @@ definitions:
...
+          description: Time between sampled points
+        image_time:
+          type: np.float64
+          unit: ms
+          m_annotations:
+            eln:
+              component: NumberEditQuantity
+              defaultDisplayUnit: ms
+          description: Time between recorded images
     My_nice_process:
```
Comment (aalbino2) [inline melt_czochralski_Dadzis/multilog_eln.archive.yaml:30]: This was just a dummy example name,  are you changing it?

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml [hunk 2]
```diff
@@ -101,31 +101,23 @@ definitions:
...
-              DAQ_6510_sensors_locations:
-                type: str
-                description: Picture of sensor locations
-                m_annotations:
-                  browser:
-                    adaptor: RawFileAdaptor
-                  eln:
-                    component: FileEditQuantity
-              DAQ_6510_sensors_list:
+              DAQ_6510:
```
Comment (aalbino2) [inline melt_czochralski_Dadzis/multilog_eln.archive.yaml:106]: The type here is pointing to a section "type: ../upload/raw/DAQ-6510.archive.yaml#Sensors_list"

I'm not sure how the section (a class, say to speak) is looking when being used as the type of a quantity (typical types here are stuff like int, float, datetime). So I will merge this PR and then make an upload to see how it looks like

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml (hunk 1 above)
Comment (arvedes) [reply melt_czochralski_Dadzis/multilog_eln.archive.yaml:30]: I still need to think about how to use it, so I just kept it as-is. I'll update it later!

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml (hunk 2 above)
Comment (arvedes) [reply melt_czochralski_Dadzis/multilog_eln.archive.yaml:106]: I tried it, it looked good. I think it was similar before. But feel free to change it from quantity to section!

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml (hunk 1 above)
Comment (aalbino2) [reply melt_czochralski_Dadzis/multilog_eln.archive.yaml:30]: Ok, nice

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml (hunk 2 above)
Comment (aalbino2) [reply melt_czochralski_Dadzis/multilog_eln.archive.yaml:106]: So what you see is a whole Sensors_list section into the DAQ_6510 quantity? 
I will try it on my local anyway, thanks!

**Comment by aalbino2** [comment]
Let's go with this PR

**Pull request merged**